### PR TITLE
tests(trace): normalize VPCSC configuration in systests

### PR DESCRIPTION
**Pull request by tseaver**
Toward #9580.

### CODE REVIEW

File: trace/tests/system/gapic/v2/test_system_trace_service_v2_vpcsc.py [hunk 1]
```diff
@@ -21,40 +21,27 @@
...
+
+
+@pytest.fixture
+def client():
+    return trace_v2.TraceServiceClient()
+
+
+@vpcsc_config.skip_unless_inside_vpcsc
+def test_batch_write_spans_w_inside(client):
+    proejct_inside = client.project_path(vpcsc_config.project_inside)
```
Comment (busunkim96) [inline trace/tests/system/gapic/v2/test_system_trace_service_v2_vpcsc.py]:
```suggestion
    project_inside = client.project_path(vpcsc_config.project_inside)
```

File: trace/tests/system/gapic/v2/test_system_trace_service_v2_vpcsc.py [hunk 2]
```diff
@@ -21,40 +21,27 @@
...
+    client.batch_write_spans(proejct_inside, [])  # no raise
```
Comment (busunkim96) [inline trace/tests/system/gapic/v2/test_system_trace_service_v2_vpcsc.py]:
```suggestion
    client.batch_write_spans(project_inside, [])  # no raise
```

File: trace/tests/system/gapic/v1/test_system_trace_service_v1_vpcsc.py [hunk 3]
```diff
@@ -21,68 +21,52 @@
...
+def test_list_traces_w_inside(client):
+    list(client.list_traces(vpcsc_config.project_inside))  # no perms issue
+
+
+@vpcsc_config.skip_unless_inside_vpcsc
+def test_list_traces_w_outside(client):
+    with pytest.raises(exceptions.PermissionDenied) as exc:
+        list(client.list_traces(vpcsc_config.project_outside))
+
+    assert exc.value.message == _VPCSC_PROHIBITED_MESSAGE
```
Comment (busunkim96) [inline trace/tests/system/gapic/v1/test_system_trace_service_v1_vpcsc.py]: Could these be changed to

```suggestion
    assert _VPCSC_PROHIBITED_MESSAGE in exc.value.message
```

Comment (tseaver) [reply trace/tests/system/gapic/v1/test_system_trace_service_v1_vpcsc.py]: e47f6657a09

File: trace/tests/system/gapic/v2/test_system_trace_service_v2_vpcsc.py (hunk 2 above)
Comment (tseaver) [reply trace/tests/system/gapic/v2/test_system_trace_service_v2_vpcsc.py]: fc23b4c83d664fba40730c9ba2cea10388797cf0

File: trace/tests/system/gapic/v2/test_system_trace_service_v2_vpcsc.py (hunk 1 above)
Comment (tseaver) [reply trace/tests/system/gapic/v2/test_system_trace_service_v2_vpcsc.py]: fc23b4c83d664fba40730c9ba2cea10388797cf0

**Pull request merged**